client/src/components/PicCarousel.js

```diff
@@ -1,49 +1,64 @@
 import Carousel from "react-bootstrap/Carousel";
 import "bootstrap/dist/css/bootstrap.min.css";
+import "./PicCarousel.css";
 export const PicCarousel = () => {
 	return (
 		<Carousel>
 			<Carousel.Item>
 				<img
 					className="d-block w-100"
-					src="https://wallpaperaccess.com/full/2560738.jpg"
+					src="https://user-images.githubusercontent.com/125136484/222224717-4856a8d7-eb25-462f-892e-c18ed3b68188.JPG"
```

Comment:
can we not have this image stored in the assets folder of the project instead of a link